.circleci/config.yml

```diff
@@ -65,6 +65,15 @@ jobs:
           at: ~/react-native-testing-library
       - run: |
           cd website && yarn install && yarn build
+  test-examples:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/react-native-testing-library
+      - run: |
+          cd examples/reactnavigation && yarn install && yarn test
+      - run: |
+          cd examples/redux && yarn install && yarn test
```

Comment:
can we have a "for-each" that would go through all examples?

Reply:
`ls -d ./examples/* | xargs -I {} bash -c "cd '{}' && npm install && npm run test"`

This lists all directory inside the examples directory and for each of them, installs dependencies and runs a test